shepherd/Timer.py

```diff
@@ -22,13 +64,47 @@ def __init__(self, timer_type, goal_name=None):
         self.goal_name = goal_name
 
     def start_timer(self, duration):
-        """Starts a new timer with the duration (seconds) and sets timer to active"""
-        pass
+        """Starts a new timer with the duration (seconds) and sets timer to active.
+           If Timer is already running, adds duration to Timer"""
+        if(self.active):
+        	queueLock.acquire()
+        	self.endTime += time
+			heapq.heapify(Timer.eventQueue)
+			queueLock.release()
+    	else:
+    		if(!Timer.running):
```

Comment:
not python syntax i think. `not Timer.running`

Reply:
Don't even put the parentheses!